Rakefile

```diff
@@ -3,4 +3,6 @@
 
 require_relative 'config/application'
 
+require 'carrierwave/orm/activerecord'
```

Comment:
This `require` shouldn't be necessary as the Rails environment will make this available.